### PR TITLE
Themes part 5

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

**Comment by montehurd** [comment]
Look like some icons disappeared:
screen shot 2017-07-12 at 11 36 02 am

**Comment by joewalsh** [comment]
@montehurd fixed in latest commit

**Comment by montehurd** [comment]
Is there a lighter gray in the theme we could use for those icons? They're much darker now. Not a big deal either way.

**Comment by montehurd** [comment]
Oh is the thin horizontal cell divider removal deliberate? Noticed while testing.

**Comment by joewalsh** [comment]
@montehurd there's not currently, could be updated in the design pass. The separators were deliberately removed across the app

**Comment by montehurd** [comment]
That article history VC animates up in a weird way... doubting it has anything to do with your PR, but thought I'd point it out in case it's quick fix.

**Comment by montehurd** [comment]
^ there's a narrow white horizontal strip at the top of it as it rises.

**Comment by montehurd** [comment]
Oh looks like the login screens aren't responding to theme changes... were those bits WIP?

**Comment by montehurd** [comment]
^ so if I hide/show the login VC after shake toggle I do see the change take effect, so may be false alarm?

**Comment by joewalsh** [comment]
@montehurd some views will only respond after you dismiss and re-present them. People won't be able to shake to change the theme in the final version, so it's expected behavior.

**Comment by montehurd** [comment]
One more quick tweak - can we kill this new gray?
screen shot 2017-07-12 at 11 59 38 am

**Comment by joewalsh** [comment]
@montehurd fixed in 6113e1e

**Comment by montehurd** [comment]
This is all great. Oh side note - I noticed the forgot pwd, change pwd and 2 factor VC's aren't themed yet, but can wait till next round if you prefer.

File: Wikipedia/Code/WMFLanguagesViewController.m [hunk 1]
```diff
@@ -79,29 +91,30 @@ - (instancetype)initWithCoder:(NSCoder *)coder {
...
     UIBarButtonItem *xButton = [UIBarButtonItem wmf_buttonType:WMFButtonTypeX target:self action:@selector(closeButtonPressed)];
     self.navigationItem.leftBarButtonItems = @[xButton];
 
-    self.tableView.backgroundColor = [UIColor wmf_settingsBackground];
-
+    self.tableView.separatorStyle = UITableViewCellSeparatorStyleNone;
     self.tableView.estimatedRowHeight = WMFOtherLanguageRowHeight;
     self.tableView.rowHeight = UITableViewAutomaticDimension;
 
     // remove a 1px black border around the search field
```
Comment (montehurd) [inline Wikipedia/Code/WMFLanguagesViewController.m]: Is this comment still relevant?

Comment (joewalsh) [inline Wikipedia/Code/WMFLanguagesViewController.m]: Nope. Neither is the code. Removed

**Comment by joewalsh** [comment]
@montehurd forgot password, 2FA, edit and anything else will be in another follow on PR

**Comment by montehurd** [comment]
Uhhg - sorry you'll have to wade through the crufty editing VCs... was hoping to update/swiftify those before anyone saw how horrible they are :/

**Comment by joewalsh** [comment]
@montehurd you're welcome to theme the edit VCs today if you feel like it :)

**Comment by montehurd** [comment]
> @montehurd you're welcome to theme the edit VCs today if you feel like it :)

hehe depends on how much progress i make on moving the footer transforms to the page lib today :)

**Pull request merged**